integration/tests/forseti_integration_testing/controls/inventory.rb

```diff
@@ -0,0 +1,426 @@
+require 'json'
+control 'inventory' do
+
+     describe "Create an inventory" do
+        subject do
+            command("forseti inventory list")
+        end
+
+        before do
+            command("forseti inventory create").result
+        end
+
+        its("stdout") { should match /SUCCESS/}
+
+        after do
+            command("forseti inventory purge 0").result
+        end
+    end
+
+    describe "Create an inventory and check database" do
+        subject do
+            command("mysql -u root --host 127.0.0.1 --database forseti_security --execute \"select count(DISTINCT resource_id) from gcp_inventory where category='resource' and resource_type = 'project';\"")
+        end
+
+        before(:context) do
+            command("forseti inventory create").result
+            command("sudo apt-get -y install mysql-client").result
+        end
+
+        # its("stdout") { should match "2" }
+        its("stderr") { should eq ""}
+
+        after do
+            command("forseti inventory purge 0").result
+        end
+    end
+
+    describe "Purge inventory" do
+        subject do
+            command("forseti inventory purge 0")
+        end
+
+        before do
+            command("forseti inventory create").result
+        end
+
+        its("stdout") { should match /purged/ }
+
+    end
+
+    describe "List inventory" do
+        subject do
+            command("forseti inventory list")
+        end
+
+        before do
+            command("forseti inventory create").result
+        end
+
+        its("stderr") { should eq ""}
+
+        after do
+            command("forseti inventory purge 0").result
+        end
+    end
+
+    describe "Get inventory" do
+        subject do
+            command("forseti inventory get #{inventory_id}")
+        end
+
+        let :inventory_id do
+            command("forseti inventory create").result
+            JSON.parse(command("forseti inventory list").stdout).fetch("id")
+        end
+
+        its("stderr") { should eq ""}
+
+        after do
+            command("forseti inventory purge 0").result
+        end
+    end
+
+    describe "Delete inventory" do
+        subject do
+            command("forseti inventory delete #{inventory_id}")
+        end
+
+        let :inventory_id do
+            command("forseti inventory create").result
+            JSON.parse(command("forseti inventory list").stdout).fetch("id")
+        end
+
+        its("stderr") { should eq ""}
+
+    end
+
+    describe "Create inventory and model" do
+        subject do
+            command("forseti model create --inventory_index_id #{inventory_id} test_model")
+        end
+
+        let :inventory_id do
+            command("forseti inventory create").result
+            JSON.parse(command("forseti inventory list").stdout).fetch("id")
+        end
+
+        its("stdout") { should match /SUCCESS/ }
```

Comment:
ditto on checking the database that the data model has been created successfully: we have the right tables, and we have the right data in the tables

Reply:
Updated test to check the database.